src/js/components/AuthorPicker.js

```
@@ -2,261 +2,129 @@
- * @param {Array}  props.selectedAuthorIds List of selected author IDs
- * @param {Function} props.onChange      Callback when authors selection changes
- * @param {string} props.buttonLabel     Optional. Custom label for the add button
- * @param {number} props.perPage         Optional. Number of authors to fetch per page
- * @param {boolean} props.showAvatars    Optional. Whether to show author avatars
+ * @param {Object} props Component props
+ * @param {Array} props.authors Available authors list
+ * @param {Array} props.selectedAuthors Currently selected authors
+ * @param {Function} props.onAuthorsChange Callback when authors change
+ * @param {number} props.maxAuthors Maximum number of authors (0 = unlimited)
  * @return {JSX.Element} Component to render
  */
```

ESLint

[error] Expected JSDoc block lines to be aligned.

(jsdoc/check-line-alignment)